blt/src/Commands/HumsciTrait.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Acquia\Blt\Custom\Commands;
+
+/**
+ * Trait HumsciTrait.
```

Comment:
This could use a better description. It is hard to tell by the trait name what the trait is doing.